symplyphysics/laws/waves/vector/phase_velocity_from_angular_velocity_and_wavevector.py

```diff
@@ -0,0 +1,74 @@
+from symplyphysics import (
+    units,
+    angle_type,
+    Symbol,
+    Quantity,
+    Vector,
+    QuantityVector,
+    validate_input,
+    validate_output,
+    scale_vector,
+    vector_magnitude,
+)
+
+# Description
+## The phase velocity of a wave is the rate at which the wave propagates in a medium.
+## It is the velocity at which the phase of one frequency component of the wave travels.
+## The phase velocity is collinear with the wavevector.
+
+# Law: v = w * k / |k|**2
```

Comment:
It seems, you are taking the direction of `k` (by dividing k vector to k norm) and a magnitude from `w / k`. We know that `k` is inverse of wavelength, so basically we divide length by time and get velocity.
Sounds quite easy to write a proof.
Not in this PR though.

Reply:
Maybe it will be better to rewrite the law as `v = (w / |k|) (k / |k|)`, to understand easier where it comes from.

As for the proof, yes, it's basically another version of $`v_{\text{phase}} = \frac {\lambda} {T}`$ (which we don't have yet, so I'll add it later), but it makes more sense to add the proof to the scalar version of this law.